src/layout/Grid/GridSummary.tsx

```diff
@@ -400,3 +406,25 @@ function CellWithLabel({ cell, columnStyleOptions, isHeader = false, headerTitle
     </CellComponent>
   );
 }
+
+function getComponentCellData(
+  node: LayoutNode,
+  displayDataProps: DisplayDataProps,
+  textResourceBindings?: ITextResourceBindings,
+) {
+  if (node && 'getDisplayData' in node.def) {
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    return node.def.getDisplayData(node as LayoutNode<any>, displayDataProps) || '-';
+  } else if (textResourceBindings && 'title' in textResourceBindings) {
+    return <Lang id={textResourceBindings.title} />;
+  } else {
+    <GenericComponent
+      node={node}
+      overrideDisplay={{
+        renderLabel: false,
+        renderLegend: false,
+        renderedInTable: true,
+      }}
+    />;
```

Comment:
@Magnusrm Whoops! A little late here, but just saw this in-editor, and it lit up with warnings. Should it have returned something instead here? :thinking: Right now these lines do nothing.